packages/babel-plugin-proposal-pattern-matching/src/index.js

```diff
@@ -24,15 +24,30 @@ export default function({ types: t }) {
     );
   }
 
+  // assert(keys.length > 0)
```

Comment:
I bet you're writing C code?

Reply:
23333, just some notations for me, I think it's OK.

Reply:
@xtuc I have improved it in the latest commit.

Reply:
Yes thanks!